modules/@apostrophecms/image/ui/apos/components/AposImageRelationshipEditor.vue

```diff
@@ -6,7 +6,6 @@
     @inactive="modal.active = false"
     @show-modal="modal.showModal = true"
     @esc="confirmAndCancel"
-    @no-modal="$emit('safe-close')"
```

Comment:
I'm glad you found a way to eliminate this, I always considered it a code wart but wasn't sure how it could be eliminated.